src/bin/vip-app.js

```diff
@@ -41,7 +42,17 @@ command( { requiredArgs: 1, format: true } )
 				error: `App ${ arg[ 0 ] } does not exist`,
 			} );
 
-			console.log( `App ${ chalk.blueBright( arg[ 0 ] ) } does not exist` );
+			const apiErrorMsg = parseApiError( err );
+
+			let errorMsg = `Unable to locate app ${ chalk.blueBright( arg[ 0 ] ) }`;
+
+			if ( apiErrorMsg ) {
+				errorMsg += ': ' + apiErrorMsg;
+			} else {
+				errorMsg += ': Invalid application or connection issue?';
```

Comment:
The question mark is going to create more confusion. We need to make it clear what might have happened and ideally what the user should do next.

It would also be helpful to output the error in some way as well so we have a hint of what might have gone wrong.

Reply:
> The question mark is going to create more confusion. We need to make it clear what might have happened and ideally what the user should do next.

I agree. I've changed the message now. In cases were no reliable error message is available (hopefully rare), we print stack trace and suggest to the user to contact VIP Support.

> It would also be helpful to output the error in some way as well so we have a hint of what might have gone wrong.

We do this, if there is anything available to print. See [here](https://github.com/Automattic/vip-cli/blob/b869a3f764451ae21a61edf4d78e3f2be27b5138/src/lib/utils.ts#L86).